tsdoc-config/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@microsoft/tsdoc-config",
+  "version": "0.12.15",
```

Comment:
This is a weird version. Is this package expected to stay in lockstep with the `tsdoc` package?

Reply:
I think that would be best, but I'll set up the version policy in a separate PR, since it may alter the CI scripts.